### PR TITLE
Add a DTR API submenu, and fix some whitespace

### PR DESCRIPTION
**Pull request by SvenDowideit**
Signed-off-by: Sven Dowideit <SvenDowideit@home.org.au>

for now, I've put it under the DTR menu - and I'll add a referring topic under the `Command and API references` section.

I can do it the other way around too - needs discussion with @rajat-docker

### CODE REVIEW

File: config.toml [hunk 1]
```diff
@@ -55,81 +55,86 @@ relativeURLs = true
...
+# HUB Menu
+#
 [[menu.main]]
 	name = "Manage image repositories"
-  identifier = "mn_docker_hub"
+	identifier = "mn_docker_hub"
 	weight = 5
 [[menu.main]]
+	name = "The Public Hub"
+	identifier = "smn_pubhub"
```
Comment (thaJeztah) [inline config.toml]: Would "The Public Hub" confuse people? Given that we refer to it as "Docker Hub" everywhere?

File: config.toml [hunk 2]
```diff
@@ -55,81 +55,86 @@ relativeURLs = true
...
 	identifier = "smn_dhe"
 	parent = "mn_docker_hub"
 	weight = 2
 [[menu.main]]
-	name = "The Public Hub"
-	identifier = "smn_pubhub"
-	parent = "mn_docker_hub"
-	weight = 1	
+	name = "DTR APIs"
+	identifier = "smn_dtrapi"
```
Comment (thaJeztah) [inline config.toml]: Should this be below "Command and API reference"? That's where all the other APIs are

Comment (SvenDowideit) [inline config.toml]: ATM, I'm keeping all the DTR things together, and will make a pointer topic in that place, but we'll discuss that with the DTR engineering team.

File: config.toml (hunk 1 above)
Comment (SvenDowideit) [inline config.toml]: I didn't change this in this PR, just moved it.

Comment (thaJeztah) [inline config.toml]: Ah! lol, you're right. hmm.. guess it should be right then (think it's a bit confusing though)

File: config.toml (hunk 2 above)
Comment (thaJeztah) [inline config.toml]: :+1:

**Comment by rajat-docker** [comment]
I think having it as a submenu under DTR makes sense to me.

**Comment by thaJeztah** [comment]
> I think having it as a submenu under DTR makes sense to me.

@rajat-docker Yeah, I'm not particularly fond of the current menu structure, but I mainly had consistency with the other API's in mind.

Anyway, I think this should be good to go, we can always change the structure if we think otherwise later on ;-)


LGTM, thanks

File: config.toml (hunk 1 above)
Comment (fredlf) [inline config.toml]: FWIW, "Public Hub" is a term that was introduced when the new docs site was implemented. I have no idea if it was ever vetted or approved by anyone in Product or the Hub team.

Comment (thaJeztah) [inline config.toml]: @fredlf you think we should change to "Docker Hub"?

Comment (fredlf) [inline config.toml]: According to Scott Johnston, VP of Product, yes, Public Hub was never vetted nor approved. It should be changed back to Docker Hub.

Comment (thaJeztah) [inline config.toml]: Thanks for checking, @fredlf, I'll create a PR :+1:

**Pull request merged**